examples/nodejs/token-vending-machine/infrastructure/test/infrastructure.test.ts

```diff
@@ -3,15 +3,14 @@
 // import * as Infrastructure from '../lib/infrastructure-stack';
 
 // example test. To run these tests, uncomment this file along with the
-// example resource in lib/infrastructure-stack.ts
+// example resource in lib/token-vending-machine-stack.ts
 test('SQS Queue Created', () => {
-//   const app = new cdk.App();
-//     // WHEN
-//   const stack = new Infrastructure.InfrastructureStack(app, 'MyTestStack');
-//     // THEN
-//   const template = Template.fromStack(stack);
-
-//   template.hasResourceProperties('AWS::SQS::Queue', {
-//     VisibilityTimeout: 300
-//   });
+  //   const app = new cdk.App();
+  //     // WHEN
+  //   const stack = new Infrastructure.TokenVendingMachineStack(app, 'MyTestStack');
+  //     // THEN
+  //   const template = Template.fromStack(stack);
+  //   template.hasResourceProperties('AWS::SQS::Queue', {
+  //     VisibilityTimeout: 300
+  //   });
```

Comment:
id just delete this file if we dont plan on using CDK integration tests. can re-add when want to add those if get around to it.

Reply:
can do